src/test/java/io/logz/LogzioSenderTest.java

```diff
@@ -67,11 +67,11 @@ public void simpleAppending() throws Exception {
         // Sleep double time the drain timeout
         Thread.sleep(drainTimeout * 1000 * 2);
 
-        assertTrue(mockListener.checkForLogExistence(token, type, loggerName, Level.INFO, message1, false, null));
-        assertTrue(mockListener.checkForLogExistence(token, type, loggerName, Level.WARN, message2, false, null));
+        assertTrue(mockListener.checkForLogExistence(token, type, loggerName, Level.INFO, message1, false, null, null));
```

Comment:
But then i get stuck with lots of overrides.. i just refactor this if I need to add anything instead of creating another method